src/text_dispatcher.erl

```diff
@@ -0,0 +1,21 @@
+-module(text_dispatcher).
+-include_lib("eunit/include/eunit.hrl").
+
+%% API
+-export([dispatch/1]).
+
+process_count() -> 4.
+
+dispatch([]) -> empty_text;
+dispatch(Text) -> dispatch(Text, process_count(), 0, string:length(Text), []).
+dispatch(_, 0, _, _, PidList) -> lists:reverse(PidList);
+dispatch(Text, ProcessesLeft, SliceStart, TextLength, PidList) ->
+  SliceLength = TextLength div ProcessesLeft,
+  Pid = spawn(graph_processor, process, [string:slice(Text, SliceStart, SliceLength)]),
+  dispatch(Text, ProcessesLeft - 1, SliceStart + SliceLength, TextLength - SliceLength, [Pid|PidList]).
+
+%% TEST
+process_count_test() ->
+  [
+    ?assert(process_count() =:= 4) % :^)
```

Comment:
:grin: add some tests to check if whole text is dispatched correctly - if you adhere to what I have written above and create a function, that prepares the list of slice tuples, you can test it if those slices cover the whole text from end to end without gaps.

Reply:
manual tests done by @me won't do? xD

Reply:
Not for "the future generations"